plugin.php

```diff
@@ -15,7 +15,7 @@
  * Author URI: https://www.constantcontact.com/
  * Text Domain: cc-woo
  * WC requires at least: 3.6.0
- * WC tested up to: 4.0.1
+ * WC tested up to: 5.8.2
```

Comment:
`WC tested up to` is the WooCommerce version that you have tested up to, not the WordPress version.
https://woocommerce.com/document/create-a-plugin/#section-10

The latest version of WooCommerce is 5.9.0, did you test against WooCommerce 5.9.0?